account.rs/src/wasm/src/account.rs

```diff
@@ -342,6 +345,55 @@ impl Account {
             .map_err(AccountError::AesGcmSiv)
     }
 
+    #[wasm_bindgen(js_name = decryptNaclUtxo)]
+    pub fn decrypt_nacl_utxo(
+        &self,
+        ciphertext: Vec<u8>,
+        commitment: Vec<u8>,
+    ) -> Result<Vec<u8>, AccountError> {
+        let nonce = GenericArray::from_slice(&commitment[0..24]);
+        let signer_pub_key = SecretKey::from(SECRET_KEY).public_key();
+        let private_key = SecretKey::from(self.encryption_private_key);
+        let nacl_box = Box::new(&signer_pub_key, &private_key);
+        let decrypted_message = nacl_box
+            .decrypt(nonce, &ciphertext[..])
+            .map_err(|_| AccountError::Generic("can't decrypt nacl".to_string()))?;
+
+        Ok(decrypted_message)
+    }
+
+    /**
+     * Encrypts utxo bytes to public key using a nonce and a standardized secret for hmac.
+     * @static
+     * @param publicKey - The public key to encrypt to.
+     * @param bytes_message - The message to be encrypted.
+     * @param commitment - The commitment used to generate the nonce.
+     * @returns The encrypted Uint8Array.
+     */
```

Comment:
Docstring syntax in Rust is a bit different. And rustdoc is not going to generate any docs for such comment syntax. It needs `///` in each line and has more markdown-like syntax. I would write it as:

```suggestion
/// Encrypts UTXO bytes to a public key using a nonce and a standardized secret for HMAC.
///
/// # Arguments
/// * `public_key` - The public key to encrypt to.
/// * `bytes_message` - The message to be encrypted.
/// * `commitment` - The commitment used to generate the nonce.
///
/// # Returns
/// The encrypted `Uint8Array`.
```